# The state of a bicycle model robot (x, y, theta, phi)
float64 x
float64 y
float64 theta
float64 phi